src/nvim/api/vim.c

```diff
@@ -33,6 +34,22 @@
 # include "api/vim.c.generated.h"
 #endif
 
+/// @see foldCreate
```

Comment:
Separate PRs would help a lot for this. Changing the folding code is fairly high-risk, so small PRs + testing is our best hope.